modules/serverless-logic/pages/cloud/operator/upgrade-serverless-operator/upgrade_1_34_0_to_1_35_0.adoc

```diff
@@ -0,0 +1,190 @@
+
+. Execute the step `1` of the upgrade for every workflow with the <<workflows_dev_profile, dev profile>>.
+. Execute the steps `1`, `2` and `3` of the upgrade for every workflow with the <<workflows_preview_profile, preview profile>>.
+. Execute the step `1` of the upgrade for every workflow with the <<workflows_gitops_profile, gitops profile>>.
+. Execute the step `1` of the <<data_index_upgrade, Data Index>> upgrade.
+. Execute the step `1` of the <<jobs_service_upgrade, Jobs Service>> upgrade.
+. Upgrade the OSL operator to the version 1.35.0 using the OpenShift management console, and wait until the new version is running, <<check_if_operator_is_updated, see>>.
+. Finalize the <<data_index_upgrade, Data Index>> upgrade by continuing from step `2`.
+. Finalize the <<jobs_service_upgrade, Jobs Service>> upgrade by continuing from step `2`.
+. Finalize the upgrade for the workflows with the <<workflows_gitops_profile, gitops profile>> by continuing from step `2`.
+. Finalize the upgrade for the workflows with the <<workflows_preview_profile, preview profile>> by continuing from step `4`.
+. Finalize the upgrade for the workflows with the <<workflows_dev_profile, dev profile>> by continuing from step `2`.
+
+[#workflows_dev_profile]
+==== Workflows with the `dev` profile
+
+Every workflow with the `dev` profile must be deleted before applying the operator upgrade to OSL 1.35.0, and re-deployed after the upgrade is completed.
+
+For every workflow `my-workflow` with the `dev` profile you must:
+
+*Pre-operator upgrade steps:*
+
+. oc delete -f <my-workflow.yaml>  -n  <target_namespace>
++
+Be sure that you have a copy of the corresponding SonataFlow CRD, as well as any other k8s resource created for that workflow, for example, the user provided ConfigMap with the application.properties in cases you are using custom properties configurations, etc.
+
+*Post-operator upgrade steps:*
+
+[start=2]
+. Re-deploy the workflow
++
+Be sure that any k8s resource for that workflow, for example, the user provided ConfigMap with the application.properties, is created before you re-deploy the workflow.
+
+[#workflows_preview_profile]
+==== Workflows with the `preview` profile
+Every workflow with the `preview` profile must be deleted before applying the operator upgrade to OSL 1.35.0, and re-deployed after the upgrade is completed.
+
+For every workflow `my-workflow` with the `preview` profile you must:
+
+*Pre-operator upgrade steps:*
+
+. If the workflow is configured to use persistence, you must backup the workflow database.
+Be sure that your database backup includes the database objects, and not only the table's information.
+
+. oc delete <my-workflow.yaml> -n <target_namespace>
++
+Be sure that you have a copy of the corresponding SonataFlow CRD, as well as any other k8s resource created for that workflow, for example, the user provided ConfigMap with the application.properties in cases you are using custom properties configurations.
```

Comment:
```suggestion
Ensure that you have a copy of the corresponding SonataFlow CRD, as well as any other k8s resource created for that workflow, for example, the user provided ConfigMap with the `application.properties` in case you are using custom properties configurations.
```